common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/MappingTest.scala

```diff
@@ -0,0 +1,227 @@
+      labels = List("a label", "another label"),
+      metadata = testImageMetadata,
+      usageRights = Some(NoRights),
+      photoshoot = Some(Photoshoot("crazy times shoot"))
+    )),
+    metadata = testImageMetadata,
+    originalMetadata = testImageMetadata,
+    usageRights = Agency(
+      supplier = "Karpow Images Inc",
+      suppliersCollection = Some("All The Metadata™"),
+      restrictions = Some("All the time")
+    ),
+    originalUsageRights = StaffPhotographer(
+      photographer = "Barton",
+      publication = "Bricking It (The Lego Times)",
+      restrictions = Some("Not before June")
+    ),
+    exports = List(
+      Crop(
+        id = Some("1234567890987654321"),
+        author = Some("me"),
+        date = Some(imageModified),
+        specification = CropSpec(
+          uri = "http://host/file",
+          bounds = Bounds(
+            x = 100, y = 100, width = 500, height = 300
+          ),
+          aspectRatio = Some("5:3"),
+          `type` = CropExport
+        ),
+        master = Some(testAsset),
+        assets = List(testAsset)
+      )
+    ),
+    usages = List(Usage(
+      id = "usage1",
+      references = List(
+        UsageReference(
+          `type` = FrontendUsageReference,
+          uri = Some(new URI("something:fancy")),
+          name = Some("frontend-use")
+        )
+      ),
+      platform = DigitalUsage,
+      media = "some-media",
+      status = PublishedUsageStatus,
+      dateAdded = Some(imageTaken),
+      dateRemoved = Some(imageModified),
+      lastModified = imageModified,
+      printUsageMetadata = Some(
+        PrintUsageMetadata(
+          sectionName = "g1",
+          issueDate = imageModified,
+          pageNumber = 1,
+          storyName = "front-page-story",
+          publicationCode = "G1M",
+          publicationName = "G1 main",
+          layoutId = Some(1234L),
+          edition = Some(1),
+          size = Some(PrintImageSize(500, 300)),
+          orderedBy = Some("Bob"),
+          sectionCode = "NEWS",
+          notes = Some("a boss story"),
+          source = Some("source information")
+        )
+      ),
+      digitalUsageMetadata = Some(DigitalUsageMetadata(
+        webUrl = new URI("https://gu.com/12345"),
+        webTitle = "Article title",
+        sectionId = "uk/news",
+        composerUrl = Some(new URI("https://composer/api/2345678987654321345678"))
+      )),
+      syndicationUsageMetadata = Some(SyndicationUsageMetadata(
+        partnerName = "friends of ours"
+      )),
+      frontUsageMetadata = Some(FrontUsageMetadata(
+        addedBy = "me",
+        front = "uk/news"
+      )),
+      downloadUsageMetadata = Some(DownloadUsageMetadata(
+        downloadedBy = "me"
+      ))
+    )),
+    leases = LeasesByMedia.build(List(
+      MediaLease(
+        id = Some("leaseA"),
+        leasedBy = Some("you"),
+        startDate = Some(imageTaken),
+        endDate = Some(imageModified),
+        access = AllowUseLease,
```

Comment:
Would making this a deny lease reduce the chance of this image flashing up in the UI?

Reply:
Not really, it’s No rights already, so will only show up when one unchecks Free to use. The best way, if possible, and should apply to Integration Tests’s image too!, would be to mock uploadTime to, like, 3 years ago.

Reply:
It's no longer NoRights, but all the dates are in 2010 so it shouldn't appear as a recent image. 🤞 

I am very happy to make this a deny lease. I suspect that if this image were to be returned to the client, the client would potentially be quite unhappy.

Reply:
Might be worth making it anodyne for the client just in case.

Reply:
Confirmed that the images don't break the client. They don't work properly, but they don't make it upset.